docs/recipes/WritingTests.md

```diff
@@ -465,8 +465,82 @@ it('passes dispatch and getState', () => {
 
 In some cases, you will need to modify the `create` function to use different mock implementations of `getState` and `next`.
 
+### Testing with Storybook
+
+Integrating [Storybook](https://storybook.js.org/) with your redux-based application is very straightforward. The main requirement is to make sure that your stories are wrapped with `Provider` so that the components have access to the store.
+
+> ##### Note on `connect`ed components
+>
+> If your components are all `connect`-based and not yet utilizing hooks, the follow example is still relevant to you, but you can always pass props into the component instead of relying on setting the `preloadedState` as shown below. There is a complete example of that approach in the [Storybook data documentation](https://www.learnstorybook.com/intro-to-storybook/react/en/data/).
+
+First, we'll add a `createStore` utility that leverages [Redux-Toolkit's](https://redux-toolkit.js.org/api/configureStore) `configureStore`.
+
+```js
+// store.js
+
+import { configureStore } from '@reduxjs/toolkit'
+import counterReducer from '../features/counter/counterSlice'
+
+// Export a store creator helper for usage by the stories
+export const createStore = options =>
+  configureStore({
+    reducer: {
+      counter: counterReducer
+    },
+    ...options
+  })
+
+export const store = createStore()
```

Comment:
It is not clear on first read that this is meant to be the `store` to be used by that app AND the `createStore` method to be used by storybook.

Maybe format it as `diff` instead of `js` to make clear that it is a recommended change to an existing file?

Reply:
Good idea. Will update this.